Automattic/jetpack
# Protect: Update pricing page

#### Description
This PR is intended to bring the Protect plugin pricing page in line with all other standalone Jetpack plugin offerings. 

#### Changes proposed in this Pull Request:

* Introduce the use of the shared [PricingTable](https://automattic.github.io/jetpack-storybook/?path=/story/js-packages-components-pricing-table--default) component on the Interstitial page.
* Adds analytics tracking for the click events
* Fixes the ProductPrice component display bug on the initial installation of the plugin

#### TODO items
* Update paid plan where applicable, when available
* Add tooltip popover content, when available

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1202920335836514
- p1HpG7-gSG-p2

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





- Checkout this branch
- Load your Jurassic Tube
- Install Protect and proceed to the pricing page
- Verify that the UI structure/styling complies with the standards set out in the Figma designs
- Verify that all hover and click events respond as expected
- Verify that the plugin continues to function as expected
